plugins/node/opentelemetry-instrumentation-mysql2/env

```diff
@@ -1,4 +1,4 @@
-export MYSQL_DATABASE=circle_database
+export MYSQL_DATABASE=otel_mysql_database
 export MYSQL_HOST=mysql
 export MYSQL_PASSWORD=secret
 export MYSQL_PORT=3306
```

Comment:
Worth to check whether this file is still needed

Reply:
I cannot find how this env file is used.
@Rauno56 seems like you added it, can you tell us if it is used anywhere?